.github/workflows/apply-version-label-issue.yml

```
@@ -0,0 +1,15 @@
+name: Apply version label to issue
+
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  add-version-label-issue:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+
+    steps:
+      - uses: gabrieldonadel/core-workflow-apply-version-label@v0.0.1
```

```suggestion
      - uses: gabrieldonadel/actions-apply-version-label@v0.0.1
```
I suggest we rename to this as it's more common across the GH actions ecosystem.

---

Sure, I've just renamed it